src/test/setup-for-puppeteer.ts

```diff
@@ -0,0 +1,9 @@
+if (process.env.CI) {
+  // set default timeout to something quite large in CI
+  jest.setTimeout(90 * 1000);
+} else {
+  jest.setTimeout(120 * 1000);
+}
+
+// export to prevent isolatedModules error
+export {};
```

Comment:
What is the purpose of this file?

Reply:
i think the idea is to separate the puppeteer setup from `src/test/setup.ts` because some of the stuff in there is unnecessary for puppeteer and potentially causing issues or at least confusing